<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="10" passed="8" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-04-25T22:45:36Z" name="Default suite" finished-at="2021-04-25T22:46:48Z" duration-ms="72269">
    <groups>
    </groups>
    <test started-at="2021-04-25T22:45:36Z" name="Default test" finished-at="2021-04-25T22:46:48Z" duration-ms="72269">
      <class name="bibliotecatestingqa.BibliotecaEscolar">
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:bibliotecatestingqa.BibliotecaEscolar@561b6512]" started-at="2021-04-25T17:45:36Z" name="beforeClass" finished-at="2021-04-25T17:45:42Z" duration-ms="5868" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method signature="cancelarregistrousuario()[pri:0, instance:bibliotecatestingqa.BibliotecaEscolar@561b6512]" started-at="2021-04-25T17:45:42Z" name="cancelarregistrousuario" finished-at="2021-04-25T17:45:48Z" duration-ms="5721" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cancelarregistrousuario -->
        <test-method signature="ingresoregistrousuariologin()[pri:0, instance:bibliotecatestingqa.BibliotecaEscolar@561b6512]" started-at="2021-04-25T17:45:48Z" name="ingresoregistrousuariologin" finished-at="2021-04-25T17:45:52Z" duration-ms="4650" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ingresoregistrousuariologin -->
        <test-method signature="iniciosesiocontraseñalnulo()[pri:0, instance:bibliotecatestingqa.BibliotecaEscolar@561b6512]" started-at="2021-04-25T17:45:52Z" name="iniciosesiocontraseñalnulo" finished-at="2021-04-25T17:45:57Z" duration-ms="4725" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- iniciosesiocontraseñalnulo -->
        <test-method signature="iniciosesioemailnulo()[pri:0, instance:bibliotecatestingqa.BibliotecaEscolar@561b6512]" started-at="2021-04-25T17:45:57Z" name="iniciosesioemailnulo" finished-at="2021-04-25T17:46:02Z" duration-ms="5059" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- iniciosesioemailnulo -->
        <test-method signature="iniciosesiondatoscorrectos()[pri:0, instance:bibliotecatestingqa.BibliotecaEscolar@561b6512]" started-at="2021-04-25T17:46:02Z" name="iniciosesiondatoscorrectos" finished-at="2021-04-25T17:46:07Z" duration-ms="4669" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- iniciosesiondatoscorrectos -->
        <test-method signature="iniciosesiondatosincorrectos()[pri:0, instance:bibliotecatestingqa.BibliotecaEscolar@561b6512]" started-at="2021-04-25T17:46:07Z" name="iniciosesiondatosincorrectos" finished-at="2021-04-25T17:46:11Z" duration-ms="4493" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- iniciosesiondatosincorrectos -->
        <test-method signature="iniciosesionnulos()[pri:0, instance:bibliotecatestingqa.BibliotecaEscolar@561b6512]" started-at="2021-04-25T17:46:11Z" name="iniciosesionnulos" finished-at="2021-04-25T17:46:16Z" duration-ms="4281" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- iniciosesionnulos -->
        <test-method signature="registrousuario()[pri:0, instance:bibliotecatestingqa.BibliotecaEscolar@561b6512]" started-at="2021-04-25T17:46:16Z" name="registrousuario" finished-at="2021-04-25T17:46:43Z" duration-ms="27014" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrousuario -->
        <test-method signature="registrousuarioobligatorio()[pri:0, instance:bibliotecatestingqa.BibliotecaEscolar@561b6512]" started-at="2021-04-25T17:46:43Z" name="registrousuarioobligatorio" finished-at="2021-04-25T17:46:46Z" duration-ms="3046" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertFalse(Assert.java:63)
	at org.testng.Assert.assertFalse(Assert.java:73)
	at bibliotecatestingqa.BibliotecaEscolar.registrousuarioobligatorio(BibliotecaEscolar.java:166)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- registrousuarioobligatorio -->
        <test-method signature="validandocaracteres()[pri:0, instance:bibliotecatestingqa.BibliotecaEscolar@561b6512]" started-at="2021-04-25T17:46:46Z" name="validandocaracteres" finished-at="2021-04-25T17:46:48Z" duration-ms="2691" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [10]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [10]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:165)
	at bibliotecatestingqa.BibliotecaEscolar.validandocaracteres(BibliotecaEscolar.java:193)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validandocaracteres -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:bibliotecatestingqa.BibliotecaEscolar@561b6512]" started-at="2021-04-25T17:46:48Z" name="afterClass" finished-at="2021-04-25T17:46:48Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
      </class> <!-- bibliotecatestingqa.BibliotecaEscolar -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
